core/main/client/browser.js

```diff
@@ -2305,6 +2317,18 @@ beef.browser = {
         }
         ;   // Microsoft Edge
 
+        if (this.isEpi()) {
+            // believe the UserAgent string for version info - until whenever
+            var epiphanyRe = /Epiphany\/(\d+)/;
+            var versionDetails = epiphanyRe.exec( beef.browser.getBrowserReportedName());
+            if (versionDetails.length > 1) {
+                return versionDetails[1];
+            } else {
+                return "UNKNOWN"; // returns from here or it may take Safari version details
+            }
```

Comment:
```javascript
            if (versionDetails.length > 1) {
                return true;
            } else {
                return false;
            }
```

Reply:
This function returns a string -- returning a boolean will break things

Top of the function:

Line: 1677
`    /**
     * Returns the type of browser being used.
     * @return: {String} User agent software and version.
     *
     * @example: beef.browser.getBrowserVersion()
     */
    getBrowserVersion: function () {
`

Bottom of the function:  line 2376.

`
        return 'UNKNOWN';				// Unknown UA
    },
`